stack.go

```diff
@@ -0,0 +1,21 @@
+package slogx
+
+import (
+	"fmt"
+	"log/slog"
+	"runtime"
+)
+
+func Stack() slog.Attr {
+	var stack string
+	pcs := make([]uintptr, 10)
+	frames := runtime.CallersFrames(pcs[:runtime.Callers(2, pcs)])
+	for {
+		frame, more := frames.Next()
+		stack += fmt.Sprintf("%s\n", frame.Function)
```

Comment:
I think this is not enough. To me, stack trace should contain not just function names, but also source file:line - just like panic output looks like. Preferably formatted in exactly same way as panic.

Also I think this is a bit too much. :smile: I mean it should exclude from output Go internals (like "runtime.goexit" etc.) just like usual panic output does.

Probably you can try to use `runtime.Stack()` instead, and then drop a couple of first lines from it output to exclude slog.Stack() itself.